region/brinstar/green/Green Hill Zone.json

```diff
@@ -352,7 +398,8 @@
           "HiJump",
           "h_canFly",
           "canWalljump",
-          "canUseFrozenEnemies"
+          "canUseFrozenEnemies",
+          "canTrickySpringBallJump"
```

Comment:
Same thing, a "canTrickySpringBallJump" would imply having Morph, so no need to wiggle.